### DISCUSSION TITLE
Conditional eps

### DISCUSSION THREAD

**Discussion by Mutaz94** [nlmixrdevelopment/nlmixr#597] [Q&A]
Hi @mattfidler 

Can we request to output 'eps' or 'err' for each record? Not the expectation as NONMEM would do? For example if one add eps on 'time' to estimate the deviation. NONMEM would estimate the value but cannot tabulate the error (as far as I know). Thanks!

**Comment by mattfidler** [comment]
I don't really know what you are asking here.  `nlmixr` outputs the `eps` or `IWRES` for each record.  There is only one `eps` estimated in `focei` at the moment.

**Comment by Mutaz94** [comment]
The idea as the following (theoretical):
Let the actual time be `t_actual` and its a function of the recorded time `t_rec`
```
t_actual_i = t_rec_i + eps_i 
```
Where eps_i ~ N(0, sigma). 

Then I can write down the analytical solution to plug in `t_actual`. 

```
C(t) = D/V * exp(-ke * t_actual)
```

Any idea if `saem` can do this job? With `eta` its going to be hard since eta change once per individual. I want more flexibility per record.

**Reply by mattfidler** [reply]
You would have to combine model times with inter-occasion availability for all time-points.  I think you could do this with NONMEM, but not yet with`nlmixr`.  Regardless I have strong concerns this would be unidentifiable and any information you get from this approach would be meaningless since you could get an infinite family of solutions that would work for the problem.
